actions/stack/create-stack/entrypoint

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+shopt -s inherit_errexit
+
+function main() {
+  local provided_secrets
+  provided_secrets=""
+
+  while [ "${#}" != 0 ]; do
+    case "${1}" in
+      --secrets)
+        provided_secrets="${2}"
+        shift 2
+        ;;
+
+      "")
+        shift
+        ;;
+
+      *)
+        echo "unknown argument \"${1}\""
+        exit 1
+    esac
+  done
+
+  args=()
+
+  # If the repository name contains 'full' then we create the stack with
+  # the --unbuffered flag to avoid hitting memory limits in github
+  # workers
+  if [[ "${GITHUB_REPOSITORY}" == *-"full"-* ]]; then
```

Comment:
I haven't seen that many actions use Github environment variables over action arguments. Any specific reason why?